action.yml

```diff
@@ -32,6 +32,11 @@ inputs:
     description: "If truthy, the token will not be revoked when the current job is complete"
     required: false
     deprecationMessage: "'skip_token_revoke' is deprecated and will be removed in a future version. Use 'skip-token-revoke' instead."
+  # Make GitHub API consigurable to support GitHub Enterprise server usecases
```

Comment:
```suggestion
  # Make GitHub API configurable to support GitHub Enterprise server use cases
```

Reply:
Maybe "...to support non-GitHub Cloud use cases"? There are [other product offerings](https://docs.github.com/en/github-ae@latest/admin/overview/about-github-ae) than GHEC and GHES.